tests/src/test/scala/org/http4s/HeadersSpec.scala

```diff
@@ -46,7 +46,7 @@ class HeadersSpec extends Http4sSpec {
         Header("Accept-Patch", ""),
         Header("Access-Control-Allow-Credentials", "")
       )
-      headers.get(`Accept-Patch`).map(_.value) must beSome("")
+      headers.get(`Access-Control-Allow-Credentials`).map(_.value) must beSome("")
```

Comment:
I'm not sure about the meaning of this test

Reply:
I think it was just making sure that we could get headers when we had a key but not a full model.  I don't think it's a great test, and what you did is fine.